### DISCUSSION TITLE
Git LFS band and Pro plan

### DISCUSSION THREAD

**Discussion by fedcal** [community/community#50994] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

With Pro plan i have unlimeted storage and bandwidth for Git LFS Data?

**Comment by CodeMaster7000** [comment]
Hello @fedcal 

In addition to the features available with GitHub Free, GitHub Pro gives:

- GitHub Support via (email)
- 3,000 GitHub Actions minutes per month
- 2 GB GitHub Packages storage
- 180 GitHub codespaces core hours per month
- 20 GB GitHub codespaces storage per month
- Repository insights graphs

Every account using Git Large File Storage receives **1 GB of free storage and 1 GB a month of free bandwidth**. You can choose to purchase an additional quota for Git LFS if your current quota is insufficient.

You can read more about this here: https://docs.github.com/en/get-started/learning-about-github/githubs-products. 

Hope this helps!
